libs/core/src/global/styles/tokens/_output/base/_semantic.scss

```diff
@@ -126,26 +127,26 @@
-  --pine-typography-body-sm-medium: var(--pine-font-weight-medium) var(--pine-font-size-body-sm)/var(--pine-line-height-body) var(--pine-font-family-inter);
-  --pine-typography-body-sm-bold: var(--pine-font-weight-bold) var(--pine-font-size-body-sm)/var(--pine-line-height-body) var(--pine-font-family-inter);
-  --pine-typography-body-sm-brand-label: var(--pine-font-weight-medium) var(--pine-font-size-body-sm)/var(--pine-line-height-body) var(--pine-font-family-greet); /* brand-label */
   --pine-typography-body-sm-brand-text: var(--pine-font-weight-medium) var(--pine-font-size-body-sm)/var(--pine-line-height-body) var(--pine-font-family-faire-sprig); /* brand-text */
   --pine-outline-focus: var(--pine-border-width-thick) solid var(--pine-color-focus-ring); /* Default focus ring */
+  --pine-typography-heading-1: var(--pine-font-weight-semi-bold) var(--pine-font-size-heading-1)/var(--pine-line-height-heading) var(--pine-font-family-heading);
+  --pine-typography-heading-2: var(--pine-font-weight-semi-bold) var(--pine-font-size-heading-2)/var(--pine-line-height-heading) var(--pine-font-family-heading);
+  --pine-typography-heading-3: var(--pine-font-weight-semi-bold) var(--pine-font-size-heading-3)/var(--pine-line-height-heading) var(--pine-font-family-heading);
+  --pine-typography-heading-4: var(--pine-font-weight-semi-bold) var(--pine-font-size-heading-4)/var(--pine-line-height-heading) var(--pine-font-family-heading);
+  --pine-typography-heading-5: var(--pine-font-weight-medium) var(--pine-font-size-heading-5)/var(--pine-line-height-heading) var(--pine-font-family-heading);
+  --pine-typography-heading-6: var(--pine-font-weight-medium) var(--pine-font-size-heading-6)/var(--pine-line-height-heading) var(--pine-font-family-heading);
+  --pine-typography-heading-caption: var(--pine-font-weight-semi-bold) var(--pine-font-size-heading-caption)/var(--pine-line-height-heading) var(--pine-font-family-heading);
+  --pine-typography-body: var(--pine-font-weight-regular) var(--pine-font-size-body-md)/var(--pine-line-height-body) var(--pine-font-family-body);
+  --pine-typography-body-medium: var(--pine-font-weight-medium) var(--pine-font-size-body-md)/var(--pine-line-height-body) var(--pine-font-family-body);
   --pine-typography-body-mono: var(--pine-font-weight-medium) var(--pine-font-size-body-md)/var(--pine-line-height-body) monospace;
+  --pine-typography-body-semi-bold: var(--pine-font-weight-semi-bold) var(--pine-font-size-body-md)/var(--pine-line-height-body) var(--pine-font-family-body);
+  --pine-typography-body-bold: var(--pine-font-weight-bold) var(--pine-font-size-body-md)/var(--pine-line-height-body) var(--pine-font-family-body);
+  --pine-typography-body-brand-label: var(--pine-font-weight-medium) var(--pine-font-size-body-md)/var(--pine-line-height-body) var(--pine-font-family-body); /* brand-label */
```

Comment:
This should be using `--pine-font-family-heading`. Take a look at line 145 on the left side.

Reply:
🦅 👁️